airbyte-cdk/java/airbyte-cdk/db-sources/src/main/java/io/airbyte/cdk/integrations/source/relationaldb/AbstractDbSource.java

```diff
@@ -88,15 +88,18 @@ public void setFeatureFlags(FeatureFlags featureFlags) {
   @Override
   @Trace(operationName = CHECK_TRACE_OPERATION_NAME)
   public AirbyteConnectionStatus check(final JsonNode config) throws Exception {
+    LOGGER.info("xiaohan - check started");
```

Comment:
I assume this is just for debugging purposes?

Reply:
reverted!